### ISSUE TITLE
Freeline plugin crashes when trying to install freeline automatically

### ISSUE THREAD

**Issue by kosiarska** [alibaba/freeline#634]
As version 2.3 Beta 2 on linux mint

````
com/android/tools/idea/gradle/AndroidGradleModel
java.lang.NoClassDefFoundError: com/android/tools/idea/gradle/AndroidGradleModel
	at utils.FreelineUtil.checkInstall(FreelineUtil.java:227)
	at actions.FreelineRunAction.actionPerformed(FreelineRunAction.java:14)
	at actions.BaseAction.actionPerformed(BaseAction.java:27)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:197)
	at com.intellij.openapi.application.TransactionGuardImpl.runSyncTransaction(TransactionGuardImpl.java:88)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransactionAndWait(TransactionGuardImpl.java:156)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:211)
	at com.intellij.openapi.actionSystem.impl.ActionButton.actionPerformed(ActionButton.java:189)
	at com.intellij.openapi.actionSystem.impl.ActionButton.performAction(ActionButton.java:155)
	at com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:326)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:795)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:627)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:387)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: com.android.tools.idea.gradle.AndroidGradleModel PluginClassLoader[com.apkfuns.plugin.freeline, 1.1.2.1]
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:64)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 42 more`
````

**Comment by lomanyong** [comment]
@pengwei1024 Is this problem caused by unsupported android studio version?

**Comment by msdx** [comment]
I met the same problem too.

**Comment by pengwei1024** [comment]
android studio2.3 beta2 和beta3 google把com.android.tools.idea.gradle.AndroidGradleModel 这个类去掉了，最近我会修好的

**Comment by pengwei1024** [comment]
The problem has been fixed in v1.1.3， you can update plugin by android studio >  plugins > Browser respsitories > Freeline Plugin

[freeline-plugin-1.1.3.jar.zip](https://github.com/alibaba/freeline/files/759616/freeline-plugin-1.1.3.jar.zip)

**Comment by kosiarska** [comment]
Can confirm, works fine now.
